frontend/components/Meetup/useFindNextMeetup.js

```diff
@@ -7,30 +7,30 @@ onMounted(() => {
       meetupsList.value = meetups
     })
   })
+  
+export function useFindNextMeetup() {
+  const nextMeetup = ref(null)
+  const today = new Date()
+  const futureMeetups = []
 
-  export function useFindNextMeetup() {
-    const nextMeetup = ref(null)
-    const today = new Date()
-    const futureMeetups = []
+  function findMeetup() {
 
-    function findMeetup() {
+    for(let meetup of MeetupsList) {
 
-      for(let meetup of MeetupsList) {
-
-        if(new Date( meetup.date ) > today) {
-          
-          futureMeetups.push( meetup )
-        }
+      if(new Date( meetup.date ) > today) {
+        
+        futureMeetups.push( meetup )
       }
-      
-      futureMeetups.sort(( a, b ) => {
-        return new Date(a.date).getTime()  -  new Date(b.date).getTime()
-      })
-
-      nextMeetup.value = futureMeetups[0]
     }
     
-    findMeetup()
+    futureMeetups.sort(( a, b ) => {
```

Comment:
```suggestion
    futureMeetups.sort((a, b) => {
```

Reply:
Hm, are you running eslint?

Reply:
yes, but I noticed that eslint does not removing all unnecessary whitespaces